components/expandable/component.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ChevronDown from '../icons/chevron-down';
+import ChevronUp from '../icons/chevron-up';
+import RadioButton, { IRadioButton } from '../forms/radio';
+
+interface IExpandable {
+  label: string;
+  expanded?: boolean;
+  radioButtonProps?: IRadioButton;
+  onExpandChange?: React.MouseEventHandler<HTMLSpanElement> | undefined;
+  content: any;
```

Comment:
Here too better to define it more specifically

Reply:
Fixed